biz.aQute.bndlib/src/aQute/bnd/build/model/EE.java

```diff
@@ -145,6 +145,17 @@ public Version getCapabilityVersion() {
 		return capabilityVersion;
 	}
 
+	/**
+	 * @return the java release number for this EE
+	 */
+	public int getRelease() {
```

Comment:
I assume this is a mapping to the javac --release target values. So perhaps a better name may be `getReleaseTarget`.

> --release release
>
> Compiles against the public, supported and documented API for a specific VM version. Supported release targets are 6, 7, 8, and 9.

Reply:
Yes, this is a better name, I fixed that